zk/txpool/pool.go

```diff
@@ -712,8 +712,8 @@ func (p *TxPool) validateTx(txn *types.TxSlot, isLocal bool, stateCache kvcache.
 		return UnsupportedTx
 	}
 
-	// Drop non-local transactions under our own minimal accepted gas price or tip
-	if !isLocal && uint256.NewInt(p.cfg.MinFeeCap).Cmp(&txn.FeeCap) == 1 {
+	// Drop transactions under our minimal accepted gas price or tip
+	if uint256.NewInt(p.cfg.MinFeeCap).Cmp(&txn.FeeCap) == 1 {
```

Comment:
will it break gasless (that come with 0 fee)?

Reply:
Thanks for the spot. I think it might, let me resolve this.

Reply:
I am making some assumptions here, but it seems like the zkevm node on your side will only allow gasless txs if they were submitted locally. In this case this check is skipped entirely, since there isnt any other checks on gasless txs on the validate tx pipeline on tx entry into the pool. I have reverted the min gasprice check to only happen for non-local txs - 3b33c6eb